tasks/README.md

```diff
@@ -25,7 +25,20 @@ Firestore Tasks collection data model
   isCollapsed: boolean,
   createdBy: string,
   assignee: string,
-  participants: [userId, userId]
+  participants: [userId, userId],
+  github: {
+    issue: {
+      status: string,
```

Comment:
can we create a model which tells us the details of `github. issue.status` values available

Some brief description on it?

Reply:
Okay, but there are only 2 values `open` and `closed` -- should I still go ahead and create a model for the same?

Reply:
It will be really helpful when we revisit this.
If a new person is working on this there also it will be really helpful 
What do you think?

Reply:
Done, yes it'll be helpful. I've included the two values beside the `status` itself.